### PR TITLE
POC Export of testkits

### PR DESCRIPTION
**Pull request by marik22312**
## Description
Exposed testkits to consumers via `react-native-ui-lib/testkit`

The main approach here was to create the testkit and test the component with it's testkit.
this way we'll have a good coverage of both the component, and the testkit in a single test file

if the component breaks, the test will fail
if there's some changes under the hood and the testkit is not updated, the tests will also fail

## Changelog
Expose initial POC of testkit for our components

### CODE REVIEW

File: testkit/package.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (marik22312) [inline testkit/package.json:1]: A small hack to make the testkits available under

`import {} from 'react-native-ui-lib/testkit'`

Comment (marik22312) [reply testkit/package.json:1]: Huge thanks to @amitdahan for helping out with this one

File: src/components/switch/index.tsx [hunk 2]
```diff
@@ -82,11 +82,14 @@ class Switch extends Component<SwitchProps> {
   getAccessibilityProps() {
     const {disabled, value} = this.props;
 
-    
+
     return {
       accessible: true,
       accessibilityRole: 'switch',
-      accessibilityStates: disabled ? ['disabled'] : value ? ['checked'] : ['unchecked'],
+      accessibilityState: {
```
[the remark this answers is not in the document]
Comment (marik22312) [reply src/components/switch/index.tsx:89]: Double checked with building & running the app locally, I see no errors regarding this change so I guess it's ok

[the remark this answers is not in the document]
Comment (ethanshar) [reply src/components/switch/index.tsx:89]: 👍 
I tried testing with Accessibility Inspector locally but it doesn't show the state value so I'll verify again on real device after we merge

**Review by ethanshar — APPROVED** [review]

**Pull request merged**